app/Http/Controllers/Web/ImportFileController.php

```diff
@@ -57,7 +57,7 @@ public function create($importType)
                 'first_name' => $user->first_name,
                 'last_name' => $user->last_name,
                 'mobile' => $user->mobile,
-                'referral' => 'user:'.$userId.',source:test,source_detail:ChompyUI',
+                'referral' => 'ads',
```

Comment:
Want to make sure I'm reading this right - this is for test imports?

Reply:
It is -- I've been meaning to update this example to include `source` and `source_details` per above, but just remove the `user` query parameter -- which would have to be constantly cleared from the text input to test other users beyond that parameter value.

Reply:
I think I'm still missing something - didn't you say you wanted to keep `source` and `source_detail`?

Reply:
Sorry! Forgot to explicitly mention that I want to update this example tracking source too (got caught up in something else)

Reply:
My turn to apologize - what is the tracking source? is that the same as `referral` here?

Reply:
Updated the names of these inputs to correspond to the column names found in the RTV CSV. Totally valid question - we do use the tracking source to determine whether the voter registration was created from a referral page, but also used it for general `source` and `source_details` regardless of its a referral -- so went with `tracking_source`